src/TeamPage/__test__/teamPage.stories.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import TeamPage from '../teamPage';
+import { Navbar } from '../../Navbar/navbar';
+import { Footer } from '../../Footer/footer';
+
+const footerActions = {
+  onLogIn: action('logging in'),
+  onLogOut: action('logging out'),
```

Comment:
If I am not mistaken, `onLogIn` and `onLogOut` had been changed to 'login` and `logout`.

Reply:
Good catch, I have updated the file.